Tests/LibCore/TestLibCoreMappedFile.cpp

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright (c) 2023, kleines Filmröllchen <filmroellchen@serenityos.org>
```

Comment:
Since so much of the file is based on the existing `TestLibCoreStream.cpp`, we should probably just list both copyrights here.